rust/src/transformations/make_stable_expr/expr_str/expr_strptime/test.rs

```diff
@@ -0,0 +1,87 @@
+use crate::domains::{AtomDomain, LazyFrameDomain};
+use crate::metrics::SymmetricDistance;
+use crate::transformations::make_stable_lazyframe;
+
+use super::*;
+
+#[test]
+fn test_make_expr_strptime() -> Fallible<()> {
+    let lf_domain = LazyFrameDomain::new(vec![
+        SeriesDomain::new("time", AtomDomain::<String>::default()),
+        SeriesDomain::new("datetime", AtomDomain::<String>::default()),
+        SeriesDomain::new("date", AtomDomain::<String>::default()),
+    ])?;
+    let lf = df!(
+        "time" => &[
+            "14:00:00".to_string(),
+            "15:00:00".to_string(),
+            "16:00:00".to_string(),
+            "16:00:00".to_string()
+        ],
+        "datetime" => &[
+            "2023-08-12 14:00:00".to_string(),
+            "2024-09-13 15:00:00".to_string(),
+            "2025-10-14 16:00:00".to_string(),
+            "2025-13-14 16:00:00".to_string()
+        ],
+        "date" => &[
+            "2023-08-12".to_string(),
+            "2024-09-13".to_string(),
+            "2025-10-14".to_string(),
+            "2025-13-14".to_string()
```

Comment:
Should you test behavior with bad data? Ill-formated, day-of-month out of range, overflow, underflow.